packages/x-license-pro/src/cli/license-cli.ts

```diff
@@ -70,14 +76,17 @@ export function licenseGenCli() {
         const licenseDetails = {
           expiryDate: new Date(new Date().getTime() + parseInt(argv.expiry, 10) * oneDayInMs),
           orderNumber: argv.order,
+          type: argv.type as LicenseType,
         };
 
         console.log(
           `Generating new license number for order ${
             licenseDetails.orderNumber
-          } with expiry date ${licenseDetails.expiryDate.toLocaleDateString()}`,
+          } with expiry date ${licenseDetails.expiryDate.toLocaleDateString()} and type "${
+            licenseDetails.type
+          }`,
```

Comment:
The quotes were not closed.

```suggestion
          }"`,
```

Reply:
Updated to v2.